code/style.css

```diff
@@ -2,6 +2,651 @@
+  width : 150px;
+  padding: 1rem;
+  margin: 0;
+  font-size: 0.8rem;
+  font-family: inherit;
+}
+
+input {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  height: 48px;
+  margin-bottom: 1rem;
+  margin-top: 0.5rem;
+  border-radius: 8px;
+  border: none;
+}
+
+.dropdown {
+  width: 100%;
+  height: 48px;
+  margin-top: 0;
+  margin-bottom: 1.5rem;
+  margin-top: 0.5rem;
+  border-radius: 8px;
+  border: none;
+}
+
+
+/* RADIOBUTTONS  */
+/* An attempt of removoving default styling... */
+
+/* .radiobutton-row input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+} */
+
+.radiobutton-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding-bottom: 2rem;
+  padding-bottom: 1rem;
+}
+
+.option {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: 0;
+}
+
+.custom-radio {
+  width: 1.125rem;
+  align-items: center;
+  margin-bottom: 0;
+  margin-top: 0%;
+  padding-right: 0.8rem;
+}
+
+.option label {
+  margin-right: 1rem;
+  margin-left: 0.5rem;
+}
+
+.option input {
+  width: 1.125rem;
+  align-items: center;
+  margin-bottom: 0;
+  margin-top: 0%;
+  padding-right: 0.8rem;
+}
+
+label {
+  font-size: 1.125rem;
+}
+
+input:valid {
+  border: 3px solid aqua;
+}
+
+/* input:invalid {
+  border: 3px solid red;
+} */
+
+
+/* FOOTER */
+
+.footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.4rem 0 0.4rem 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  background-color: #1a1a1a;
+  opacity: 70%;
+}
+
+.technigo-logo {
+  width: 40px;
+}
+
+.github-logo {
+  width: 40px;
+  height: 40px;
+}
+
+.technigo {
+  display: flex;
+  flex-direction: row;
+  margin-left: 1rem;
+}
+
+.github {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 1rem;
+}
+
+
+.footer p {
+  margin-left: 1rem;
+  margin-right: 0.5rem;
+  font-size: 14px;
+  color: #fff;
+  margin-bottom: 0;
+}
+
+.footer a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+
+/* MEDIA QERIES  */
+
+@media (min-width: 768px) and (max-width: 991px) {
+
+  h1 {
+    font-size: 140px;
+  }
+
+  .company-logo {
+    width: 96px;
+    height: auto;
+  }
+
+  .hero-image {
+    height: 550px;
+  }
+
+  .hero-text {
+    padding-bottom: 4rem;
+    padding-top: 8rem;
+  }
+
+  .sectionHeading {
+    font-size: 2rem;
+  }
+
+  .preamble {
+    margin-left: 4rem;
+    margin-right: 4rem;
+  }
+
+  .info-text {
+    margin-left: 4rem;
+    margin-right: 4rem;
+  }
+
+  .jes-img {
+    width: 140px;
+    height: 140px;
+  }
+  
+  .contact p {
+    font-size: 1.125rem;
+    margin-left: 1.5rem;
+  }
+
+  .quote {
+    margin-left: 4rem;
+    margin-right: 4rem;
+  }
+  
+  .accordion {
+    padding: 0;
+    margin-left: 4rem;
+    margin-right: 4rem;
+  }
+
+  .form {
+    margin-top: 2rem;
+    margin-left: 4rem;
+    margin-right: 4rem;
+    margin-bottom: 4rem;
```

Comment:
You can reduce these four rows by putting each value in the same property.

margin: 2rem 4rem 4rem;